Tools/peg_generator/pegen/c_generator.py

```diff
@@ -26,34 +26,58 @@
 
 EXTENSION_PREFIX = """\
 #include "pegen.h"
+
 """
+
 EXTENSION_SUFFIX = """
 static PyObject *
 parse_file(PyObject *self, PyObject *args, PyObject *kwds)
 {
     static char *keywords[] = {"file", "mode", NULL};
     const char *filename;
     int mode = %(mode)s;
-
     if (!PyArg_ParseTupleAndKeywords(args, kwds, "s|i", keywords, &filename, &mode))
         return NULL;
     if (mode < 0 || mode > %(mode)s)
         return PyErr_Format(PyExc_ValueError, "Bad mode, must be 0 <= mode <= %(mode)s");
-    return run_parser_from_file(filename, (void *)start_rule, mode, reserved_keywords, %(n_keyword_lists)s);
+
+    PyArena *arena = PyArena_New();
+    if (arena == NULL)
+        return NULL;
```

Comment:
Nit: PEP 7

Reply:
(FWIW I just pushed a commit to the 'pegen' branch in this repo that adds the missing '.clang-format' file so at least in Tools/peg_generator you can use 'make format-c' to fix the formatting of real .c files. :-)